_data/simple-icons.json

```diff
@@ -16642,6 +16642,11 @@
             "hex": "007ACC",
             "source": "https://commons.wikimedia.org/wiki/File:Visual_Studio_Code_1.35_icon.svg"
         },
+        {
+            "title": "visx",
+            "hex": "FF1231",
+            "source": "https://airbnb.io/visx/"
```

Comment:
I can't see the SVG source here - am I missing something @uncenter?

Reply:
Yeah it is a bit roundabout to access it:

![Screenshot 2024-06-07 at 05 45 26 (Arc)](https://github.com/simple-icons/simple-icons/assets/47499684/24175461-8ed0-4642-9846-7d00644095af)